### ISSUE TITLE
Nickname reset

### ISSUE THREAD

**Issue by masterchivo** [BernardoGiordano/PKSM#46]
Edited pokemon nickname is reseted after evolution

**Comment by RealRexen** [comment]
I don't think that's possible unless you give us more info. As the game does it this with foreign pokemon. E.g if I got a JPN Growlithe and evolved it, its name will display as Arcanine and not its Japaneses name

**Comment by jjaldridge2009** [comment]
I experienced this issue as well. Something is not being flagged right when a nickname is set using PKSM.

3 different Pokemon did this. Here are the steps followed with each of them:

1. Bred Pokemon in X, gave it no nickname.
2. Edited Pokemon in PKSM, added nickname.
3. Went back into X, levelled up to evolve, nickname disappeared.

In one case, I had to transfer my Cubone to Sun\Moon, so I could evolve into Alolan Marowak.

I used Pokemon Bank to perform the transfer. Evolved the Cubone in Pokemon Sun. and his nickname disappeared. He was just Marowak after that.
